### ISSUE TITLE
Solve informational problems

### ISSUE THREAD

**Issue by fazo96** [fazo96/ipfs-boards#74]
- [x] it is not clear what are the goals of the project in terms of what kind of platform we want to create: we want to create a platform to share public data easily that values accessibility and independence. Every detail about how stuff works, what can and can't be done and the reccomanded use cases should be clear
- [x] ~~we need the new name~~ in progress: #24 
- [ ] archival / deletion problem (starting from discussion in #24) maybe mention it in FAQs.
- [ ] the get started part should be overhauled to be easier and mentioned in frontpage
- [x] the demo should load the fonts fine using https
- [x] discussion board --> social network, content sharing platform

Some notes:

This platform focuses on:
- **Share your data**: upload **documents**, **media** and **files and folders**
- **Support other users**: **Star** their stuff #85 
- **Build a Community**: invite users to your boards, or join other's boards

This should be reflected on the landing page.
